src/pyspiffe/workloadapi/jwt_source.py

```diff
@@ -0,0 +1,36 @@
+"""
+This module defines the source for JWT Bundles and SVIDs.
+"""
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+from pyspiffe.bundle.jwt_bundle.jwt_bundle import JwtBundle
+from pyspiffe.spiffe_id.trust_domain import TrustDomain
+from pyspiffe.svid.jwt_svid import JwtSvid
+
+
+class JwtSource(ABC):
+    """Source of JWT-SVIDs and JWT bundles maintained via the Workload API."""
+
+    @abstractmethod
+    def get_jwt_svid(self) -> JwtSvid:
+        """Returns an JWT-SVID from the source."""
+        pass
+
+    @abstractmethod
+    def get_bundle_for_trust_domain(
```

Comment:
I think we should review the name of this method and get an agreement on how to name it.
The `JwtBundleSet` use `get()` instead of `get_bundle_for_trust_domain()` but then all the methods from the x509 side use the latter.
I remember having a conversation to define that `JwtBundleSet` should have a `get()` method because is pretty much a dictionary keyed by Trust Domain. I still prefer the `get()` name but I would like to be consistent on this name so if you think that the name here should remains as `get_bundle_for_trust_domain()` let's rename the method on `JwtBundleSet`. What do you think?

Reply:
I  think you are correct. I will rename it.. I guess I forgot that conversations :S...:)

Thanks for catching this!

Reply:
Now that I see the code again, it feels a little confusing the name `get`, as the source not only provides bundles but also jwt-svids, it seems to me that something like `get_bundle` or `get_jwt_bundle` would clarify better the purpose of the method. Sorry for bringing this up now.